docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc

```diff
@@ -0,0 +1,208 @@
+// Use these for links to issue and pulls. 
+:kib-issue: https://github.com/elastic/kibana/issues/
+:kib-pull: https://github.com/elastic/kibana/pull/
+:agent-issue: https://github.com/elastic/beats/issues/
+:agent-pull: https://github.com/elastic/beats/pull/
+:fleet-server-issue: https://github.com/elastic/beats/issues/fleet-server/
+:fleet-server-pull: https://github.com/elastic/beats/pull/fleet-server/
+
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-7.16.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 7.16.0 relnotes
+
+[[release-notes-7.16.0]]
+== {fleet} and {agent} 7.16.0
+
+Review important information about the {fleet} and {agent} 7.16.0 releases.
+
+//[discrete]
+//[[breaking-changes-7.16.0]]
+//=== Breaking changes
+
+//Breaking changes can prevent your application from optimal operation and
+//performance. Before you upgrade, review the breaking changes, then mitigate the
+//impact to your application.
+
+//[discrete]
+//[[breaking-PR#]]
+//.Short description
+//[%collapsible]
+//====
+//*Details* +
+//<Describe new behavior.> For more information, refer to {kibana-pull}PR[#PR].
+
+//*Impact* +
+//<Describe how users should mitigate the change.> For more information, refer to {fleet-guide}/fleet-server.html[Fleet Server].
+//====
+
+//[discrete]
+//[[known-issues-7.16.0]]
+//=== Known issues
+
+//[[known-issue-issue#]]
+//.Short description
+//[%collapsible]
+//====
+
+//*Details* 
+
+//<Describe known issue.>
+
+//*Impact* +
+
+//<Describe impact or workaround.>
+
+//====
+
+[discrete]
+[[enhancements-7.16.0]]
+=== Enhancements
+
+Fleet::
+* Adds prompt for users to add an agent if they add an integration to an agent policy with no agents {kib-pull}114830[#114830]
+* Allow users with read access to view Integrations app {kib-pull}113925[#113925]
+* Removes enterprise license requirement for custom registry URL {kib-pull}113858[#113858]
+* Adds "Keep Policies up to Date" functionality for integrations {kib-pull}112702[#112702]
+* Allow packages to specify index privileges {kib-pull}112397[#112397]
+* Allow Integrations browse page to filter on descriptions {kib-pull}111649[#111649]
+* Support automatic upgrades of package policies when updating integrations {kib-pull}108269[#108269]
+
```

Comment:
```suggestion
* Support automatic upgrades of package policies when updating integrations {kib-pull}108269[#108269]
* Allow preconfiguration of alternative {es} outputs {kib-pull}111002[#111002]
```

Reply:
@nchaulet You set `release_notes:skip` on the PR. Just want to confirm with you that it's OK to include this in teh release notes for Fleet/Agent. If we add it here, we should probably add it to Kibana relnotes, too. WDYT?

Reply:
So I'm going to include this line in the release notes. @nchaulet If you think it should not be included, please just let me know. I want to get this merged ASAP (can always remove the line afterwards).